core/src/components/item/item.scss

```diff
@@ -372,18 +372,6 @@ button, a {
 
 ::slotted(ion-label:not([slot="end"])) {
   flex: 1;
-
-  // Setting width to min-content allows the label to
-  // shrink and wrap its text instead of moving to its
-  // own row if there are slotted elements next to it
-  width: min-content;
-
-  /**
-   * We allow labels in the default
-   * slot to grow. However, we do not
-   * want them to grow indefinitely.
-   */
-  max-width: 100%;
```

Comment:
Since the start/end slots no longer wrap to their own lines, this code isn't needed